asv.conf.json

```diff
@@ -11,8 +11,7 @@
     "html_dir": ".asv/html",
     "build_cache_size": 8,
     "build_command": [
-      "python -m pip install 'setuptools_scm[toml]>=4,<6' milksnake",
-      "python setup.py build",
+      "python -m pip install 'setuptools_scm[toml]>=4,<6' milksnake maturin",
```

Comment:
still keeping `setuptools` and `milksnake` around for comparisons with older versions